src/front/ticketer/lib/backend_communication/model/user.dart

```diff
@@ -15,6 +15,16 @@ class User {
         'email': email,
         'password': password,
       };
+
+  factory User.fromJson(Map<String, dynamic> json) {
+    return User(
+        json['firstName'],
+        json['lastName'],
+        json['birthDate'],
+        json['email'],
+        ""
```

Comment:
Please add comment explaining that User fetched from backend dose not have a password. And make password an nullable filed in that case

Reply:
This is the most pressing issue @faliszewskii

Reply:
Addressed. Maybe